New Projects added

AnshulGoyal589 opened the pull request:
I have added TO-DO-LIST with many features such as local storage , drag and drop , varoius themes , user friendly .
and secondly I have added a snake game made using CANVAS API which user would love to play.
Closes issue #1 

Please accept my PR under hactoberfest 2023.

sree-hari-s commented on Projects/README.md:
It would be better to have separate readme files inside the folders as the Projects folder will contain a lot of projects so just the name would be better here and description in the respective folders

AnshulGoyal589 replied:
okay got it

sree-hari-s commented:
This repository is exclusive for python projects. Please look for other repos. You can contribute the documentation part. Splitting it into respective folders

AnshulGoyal589 commented:
@sree-hari-s  I have removed my non-python projects and updated the readme files as per the format you said.
If still there is an issue in anything, please let me know.

sree-hari-s commented:
LGTM, Please leave a ⭐ to this repo and keep contributing.
